Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:14.04
+MAINTAINER Anuvrat Parashar "anuvrat@anuvrat.in"
+
+RUN apt-get update && apt-get -y upgrade && apt-get install -y git python2.7 python-pip python-dev postgresql-server-dev-all
+ADD requirements.txt /srv/requirements.txt
```

Comment:
could be done, sure.
btw, are there package mentioned in requirements-dev.txt that would be essential for running the project inside docker?